pkg/ui/console/client.go

```diff
@@ -478,7 +478,7 @@ func (c Client) DescribePolicies(ctx context.Context, policySource string) error
 
 func (c Client) ValidatePolicy(ctx context.Context, policySource string) (diags diag.Diagnostics) {
 	defer printDiagnostics("", &diags, viper.GetBool("redact-diags"), viper.GetBool("verbose"))
-	policyToValidate, err := FilterPolicies(policySource, c.cfg.Policies)
+	policyToValidate, err := FilterPolicies(policySource)
```

Comment:
nit: not sure filter policies is required anymore, as it will always just wrap the policy source after detect, maybe the name is confusing.

Reply:
Not sure what to name it - It does "parse path" + "detect" and returns a policy struct.. 
"ParseAndDetct" maybe?